docs/CLIExamples.rst

```diff
@@ -0,0 +1,304 @@
+=================
+CLI Examples
+=================
+
```

Comment:
Please give a brief introduction at the top of the page.  Just a paragraph or two about the CLI and how it can help researchers or the generally curious.  It might be good to include Apple/Linux/Windows specific notes here too.